bin/http_server_spec

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -ex
+gem install bundler
+git clone https://github.com/tpetersen0308/http-server-spec.git
```

Comment:
does this work if it already exists?

Reply:
I'm not sure I understand. If this is being run by the CI, then won't it always need to be cloned anew?

Reply:
Okay, so the intent is for this to be a ci-only script... 👍 Maybe rename to `bin/ci`?

Reply:
Ah right.. thought this was how to run the spinach tests

Reply:
Ahh got it. Okay, I'll move it into a CI directory. Thanks!